cabal-install/Distribution/Client/CmdConfigure.hs

```diff
@@ -83,7 +86,13 @@ configureAction (configFlags, configExFlags, installFlags, haddockFlags)
     baseCtx <- establishProjectBaseContext verbosity cliConfig
 
     -- Write out the @cabal.project.local@ so it gets picked up by the
-    -- planning phase.
+    -- planning phase. If old config exists, then print the contents
+    -- before overwriting
+    exists <- doesFileExist "cabal.project.local"
+    when exists $ do
+        notice verbosity "'cabal.project.local' file already exists. Now overwriting it."
+        contents <- readFile "cabal.project.local"
```

Comment:
Thanks! I wasn't aware of the `LOCALE` issues with `readFile / writeFile`. I have updated the PR with `copyFile`.